region/norfair/east/Speed Booster Hall.json

```diff
@@ -112,7 +112,8 @@
           "Wave",
           "Plasma"
         ]},
```

Comment:
this one doesnt have a reset room. I havent been paying close attention to if others are missing it, but I do wonder if this room might need a reset specifically at the top door, to prevent coming in the right and needing to dodge the spikes and lava pits. Plus the chance of having speed not being able to do this because of the rising lava